main/src/main/resources/report.html

```diff
@@ -265,14 +265,14 @@ <h4>Feed Info</h4>
                 <hr />
                 <dl>
                     <div th:each="info: ${metadata.feedInfo}">
-                        <dd th:text="${info.key} + ':'" />
-                        <dt>
+                        <dd th:if="${info.key != 'Service Window Start' and info.key != 'Service Window End'}" th:text="${info.key} + ':'" />
```

Comment:
A really small thing and feel free to ignore, but can we change the code to have this test done only once?

Reply:
good catch! It's fixed now!